mountpoint-s3/scripts/network_performance.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Please install 'jq' and 'aws-cli' v2 before execute the script.
+# The purpose of this script is to generate a json file contains network throughput numbers of each EC2 instance types.
+
+version_file_prefix=network_performance_version_
+version_file_suffix=.json
+result_file=network_performance.json
+mktmp=/tmp/mktmp
+version=$( date '+%F' )
+
+# query
+query_ec2_instance_network_throughput="aws ec2 describe-instance-types
+--filters \"Name=instance-type,Values=*\"
+--query \"InstanceTypes[].[InstanceType, NetworkInfo.NetworkPerformance]\"
+--output json"
```

Comment:
We need to run this explicitly against `us-east-1`.

Reply:
I don't quit get that. You mean asign the region with the data fetched from json?